pkg/inject/sidecar_test.go

```diff
@@ -58,6 +58,39 @@ func TestInjectSidecarWithEnvVars(t *testing.T) {
 	assert.Equal(t, "jaeger,b3", dep.Spec.Template.Spec.Containers[0].Env[1].Value)
 }
 
+func TestInjectSidecarWithEnvVarsK8sAppName(t *testing.T) {
+	jaeger := v1.NewJaeger("TestInjectSidecarWithEnvVars")
```

Comment:
For the future, we could replace the instance name in tests with a random value, as this is something we always forget to change after a copy/paste... Or just have them all have the same value.  The purpose of having a unique name per test is to avoid clashes in parallel tests, but we don't have such a scenario today.

Reply:
Sorry, I'm usually the one that falls into this trap :(

Reply:
I do it all the time as well.